AppService/README.md

```diff
@@ -10,6 +10,7 @@ Also use [`Azure/appservice-settings`](https://github.com/Azure/appservice-setti
 
 * [Checkout](https://github.com/actions/checkout) Checkout your Git repository content into Github Actions agent.
 * Authenticate using [Azure Web App Publish Profile](https://github.com/projectkudu/kudu/wiki/Deployment-credentials#site-credentials-aka-publish-profile-credentials) or using [Azure Login](https://github.com/Azure/login)
+  * **Note**: For publish profile auth as of October 2020, Linux web apps will need the app setting `WEBSITE_WEBDEPLOY_USE_SCM` set to `true`. This requirement will be removed in the future.
```

Comment:
I think you just need to make it clear that the app setting change needs to happen BEFORE downloading the profile because that controls which endpoint they get in the file

Reply:
Updated the text!